docs/extensions/fares-v2.md

```diff
@@ -0,0 +1,82 @@
+  </a>
+  
+# GTFS-Fares v2
+
+Fares v2 is a GTFS Schedule extension project that aims to address the limitations of <a href="/schedule/examples/fares-v1">Fares v1</a>.
+
+The main concepts that Fares v2 plans to represent are
+
+- Fare products (e.g tickets and passes)
+- Rider categories (e.g seniors and children)
+- Fare payment options (e.g bank card and credit card)
+- Fare capping
+
+These concepts will allow data producers to model zone-based, time-dependent, and inter-agency fares. This extension project is being adopted in iterations. 
+
+Currently, the officially adopted base implementation of Fares v2 works for the following use cases:
+
+- Defining a transit fare
+- Describing service locations in the same fare zone
+- Creating rules for single leg journeys
+- Creating rules for transfers
+
+You can see <a href="/schedule/examples/fares-v2" target="_blank">examples here</a> that show what can be modelled using the official Fares v2 base implementation.
+
+Producers may implement Fares v2 in the same dataset with Fares v1, since there is no technical conflict between the two. Consumers can choose which version to use independent of the other. With adoption and sufficient endorsement of Fares v2, Fares v1 may be deprecated in the future.
+
+<a class="button no-icon" href="https://share.mobilitydata.org/gtfs-fares-v2" target="_blank">See the full proposal</a>
+
+## In Progress Fares v2 Work
+
+The GTFS community is currently working on finalizing the <a href="https://share.mobilitydata.org/fare-containers-to-fare-payment-types-proposal" target="_blank">fare_payment_options.txt (formerly fare_containers.txt)</a> proposal. 
+    
+Items currently in discussion include:
+
+- Creating a fare_payment_options_groups.txt file
+- Aligning on what types should be included in the enumeration for `fare_payment_option_type`
+
+<a class="button no-icon" href=https://share.mobilitydata.org/slack>Join #gtfs-fares on Slack</a><a class="button no-icon" href=https://www.eventbrite.ca/e/specifications-discussions-gtfs-fares-v2-monthly-meetings-tickets-522966225057>See the meeting schedule</a><a class="button no-icon" href=https://docs.google.com/document/d/1d3g5bMXupdElCKrdv6rhFNN11mrQgEk-ibA7wdqVLTU/edit>See meeting notes</a>
+
+
+## First Adopters
+
+🎉 Shoutout to Fares v2 first adopters! At least 1 data producer and 1 consumer must commit to implementing an experimental feature before a public vote is opened to add it to the official specification. These organizations invest a large amount of time and energy into experimental changes to make sure that GTFS continues to evolve.
+
+For the adopted base implementation, first adopters were
+
+- Producers: <a href="https://www.interline.io/" target="_blank">Interline</a>,<a href="https://www.mta.maryland.gov/developer-resources" target="_blank">Maryland Department of Transportation</a>,<a href="https://dot.ca.gov/cal-itp/cal-itp-gtfs" target="_blank">Cal-ITP</a>
```

Comment:
Was Maryland DOT producing as we were designing the extension?

Reply:
Looks like it based on the vote opening announcement: https://github.com/google/transit/pull/286#issuecomment-1080716109